test/lightning_web/live/workflow_live/edit_test.exs

```diff
@@ -2267,7 +2267,17 @@ defmodule LightningWeb.WorkflowLive.EditTest do
         ~p"/projects/#{project.id}/w/#{workflow.id}?#{[m: "expand", s: job.id]}"
       )
 
-      assert render(view) =~ "Workflow saved and synced to GitHub successfully."
+      assert render(view) =~ "Workflow saved and sync requested. Check the"
+
+      link_to_actions =
+        "https://www.github.com/" <> repo_connection.repo <> "/actions"
+
+      assert has_element?(
+               view,
+               ~s{#flash [href="#{link_to_actions}"][target="_blank"]},
```

Comment:
This test is failing, perhaps because we don't have a `#flash` element anymore, since the unique ids changes we use this to find flash messages: `[data-flash-kind='info']`.